src/Migrations/Version20200614133313.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buddy\Repman\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20200614133313 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql("INSERT INTO config (key, value) VALUES ('local_authentication', 'login_and_registration')");
+        $this->addSql("INSERT INTO config (key, value) VALUES ('oauth_registration', 'enabled')");
+        $this->addSql("DELETE FROM config WHERE key = 'user_registration'");
```

Comment:
What about instances where this option was already used, we need to consider this in migration and set:
- [ ] if `user_registration` was enabled then leave as now
- [ ] if `user_registration` was disabled then `user_registration` = `disabled` and `local_authentication` = `login_only`

correct me if I'm wrong

Reply:
That was my point. Should we try to gracefully upgrade? And what about downgrade?

Reply:
What you mean by `gracefully upgrade`?
In `down` migration we check `local_authentication` if is `login_only` and the set `user_registration` to `disabled` (otherwise `enabled`

Reply:
OK that makes sense. I will fix that